checkov/cloudformation/cfn_utils.py

```diff
@@ -200,8 +201,16 @@ def _parse_file(file):
         try:
             template, template_lines = parse_result
             if isinstance(template, DictNode) and isinstance(template.get("Resources"), DictNode):
-                definitions[path] = template
-                definitions_raw[path] = template_lines
+                template_resources = template.get("Resources")
+                is_valid = True
+                for resource in template_resources.values():
+                    if 'Properties' not in resource or (not isinstance(resource['Properties'], DictNode)):
+                        out_parsing_errors.update({file: 'Resource Properties is not a dictionary'})
```

Comment:
additionally it would be good to add a log message here with details about which file and the resource name.